sdk/sqlmanagement/Azure.ResourceManager.Sql/tests/SessionRecords/ElasticPoolTests/ElasticPoolApiTests.json

```diff
@@ -0,0 +1,1463 @@
+{
```

Comment:
Please remove this folder, recordings should have been migreated

Reply:
removed the SessionRecords folder